dist/windows/config.nsh

```diff
@@ -169,10 +136,9 @@ RequestExecutionLevel user
 !define MUI_PAGE_CUSTOMFUNCTION_LEAVE un.check_instance
 !insertmacro MUI_UNPAGE_COMPONENTS
 !insertmacro MUI_UNPAGE_INSTFILES
-
 !insertmacro MUI_RESERVEFILE_LANGDLL
-ReserveFile "${NSISDIR}\Plugins\x86-unicode\FindProcDLL.dll"
-ReserveFile "${NSISDIR}\Plugins\x86-unicode\UAC.dll"
```

Comment:
Are you sure that these 2 lines aren't needed?

Reply:
The installer seems to work correctly without them.

Reply:
> The installer seems to work correctly without them.

Well, I finally figured out its true purpose. Everything should be OK now.